migrations/008.do.daily-fil-stats.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE daily_fil (
```

Comment:
To me, `daily_fil` isn't self explanatory. And what will this table track? Is it the FIL sent to addesses, or when we increased their balance in the smart contract?

Reply:
IIUC, we want to track rewards transferred every Monday from "scheduled rewards" to participant wallets. 

My proposals for a better table name:
- `daily_transfers` 
- `daily_reward_transfers`

We are using the `daily_` prefix in other table names to communicate that these tables store data with a daily granularity (i.e. they all use a `day DATE NOT NULL` column as the timestamp).